GetVisibleServicesByScope/handler.ts

```diff
@@ -0,0 +1,81 @@
+import * as t from "io-ts";
+
+import * as express from "express";
+
+import { BlobService } from "azure-storage";
+
+import {
+  IResponseErrorInternal,
+  IResponseSuccessJson,
+  ResponseErrorInternal,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+
+import {
+  VISIBLE_SERVICE_CONTAINER,
+  VisibleService
+} from "io-functions-commons/dist/src/models/visible_service";
+
+import { ServiceId } from "io-functions-commons/dist/generated/definitions/ServiceId";
+import { getBlobAsObject } from "io-functions-commons/dist/src/utils/azure_storage";
+import { wrapRequestHandler } from "io-functions-commons/dist/src/utils/request_middleware";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+
+const ServicesByScope = t.partial({
+  LOCAL: t.array(ServiceId),
+  NATIONAL: t.array(ServiceId)
+});
+
+export type ServicesByScope = t.TypeOf<typeof ServicesByScope>;
+
+type IGetVisibleServicesByScopeHandlerRet =
+  | IResponseSuccessJson<ServicesByScope>
+  | IResponseErrorInternal;
+
+type IGetVisibleServicesByScopeHandler = () => Promise<
+  IGetVisibleServicesByScopeHandlerRet
+>;
+
+/**
+ * Returns all the visible services (is_visible = true) grouped by its scope.
+ */
+export function GetVisibleServicesByScopeHandler(
+  blobService: BlobService,
+  servicesByScopeBlobId: NonEmptyString
+): IGetVisibleServicesByScopeHandler {
+  return async () => {
+    const errorOrMaybeVisibleServicesByScopeJson = await getBlobAsObject(
+      ServicesByScope,
+      blobService,
+      VISIBLE_SERVICE_CONTAINER,
+      servicesByScopeBlobId
+    );
+    return errorOrMaybeVisibleServicesByScopeJson.fold<
+      IGetVisibleServicesByScopeHandlerRet
+    >(
+      error =>
+        ResponseErrorInternal(
+          `Error getting visible services by scope list: ${error.message}`
+        ),
+      maybeVisibleServicesByScopeJson => {
+        return ResponseSuccessJson(
+          maybeVisibleServicesByScopeJson.getOrElse({})
```

Comment:
I don't know the context, but why it's not a 404 when no blob is found?

Reply:
It's a good point. cc @gunzip @francescopersico